schemas/system/domains.schema

```diff
@@ -0,0 +1,8 @@
+{
+  IdStatus: Enum('Preallocated', 'Initialization', 'Actual', 'Historical'),
+  StorageKind: Enum('Master', 'Cache', 'Backup', 'Replica'),
+  CategoryRealm: Enum('Global', 'System', 'Server'),
+  CategoryFamily: Enum('Registry', 'Dictionary', 'Table', 'Log', 'View'),
+  UserAccess: Enum('Read', 'Insert', 'Update', 'Delete', 'Execute', 'Audit'),
```

Comment:
What does 'audit' imply here?

Reply:
`Audit` is a permission to view user access log, all changes of given `Category`, optionally limited by `Catalog` and `Subdivision`